specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1967 @@
+            "$ref": "#/definitions/DataTable"
+          }
+        }
+      }
+    },
+    "KeyValuePair": {
+      "description": "Information about the extracted key-value pair.",
+      "type": "object",
+      "required": [
+        "key",
+        "value",
+        "confidence"
+      ],
+      "properties": {
+        "label": {
+          "description": "A user defined label for the key/value pair entry.",
+          "type": "string"
+        },
+        "key": {
+          "description": "Information about the extracted key in a key-value pair.",
+          "$ref": "#/definitions/KeyValueElement"
+        },
+        "value": {
+          "description": "Information about the extracted value in a key-value pair.",
+          "$ref": "#/definitions/KeyValueElement"
+        },
+        "confidence": {
+          "description": "Confidence value.",
+          "$ref": "#/definitions/Confidence"
+        }
+      }
+    },
+    "KeyValueElement": {
+      "description": "Information about the extracted key or value in a key-value pair.",
+      "type": "object",
+      "required": [
+        "text"
+      ],
+      "properties": {
+        "type": {
+          "$ref": "#/definitions/KeyValueType"
+        },
+        "text": {
+          "description": "The text content of the key or value.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of the key or value.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "elements": {
+          "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this key or value.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        }
+      }
+    },
+    "KeyValueType": {
+      "type": "string",
+      "description": "Semantic data type of the key value element.",
+      "enum": [
+        "string",
+        "selectionMark"
+      ],
+      "x-ms-enum": {
+        "name": "KeyValueType",
+        "modelAsString": true
+      },
+      "x-nullable": false
+    },
+    "ElementReference": {
+      "description": "Reference to a line, word or selection mark.",
+      "type": "string"
+    },
+    "SelectionMark": {
+      "description": "Information about the extracted selection mark.",
+      "type": "object",
+      "required": [
+        "boundingBox",
+        "confidence",
+        "state"
+      ],
+      "properties": {
+        "boundingBox": {
+          "description": "Bounding box of the selection mark.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Confidence value.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "state": {
+          "description": "State of the selection mark.",
+          "type": "string",
+          "enum": [
+            "selected",
+            "unselected"
+          ]
```

Comment:
Are there other values that should be consider or that will be added in the future?
If not, I recommend adding `"modelAsString": false`

Reply:
Although the 2.1-preview.1 swagger was merged, I still have this question, so leaving question

Reply:
@ramparab could you confirm if there are other values that could be added into this enum in the future?

Reply:
We may introduce "indetermined" in the future, but unlikely any others.  I would agree with using modelAsString=false.